### ISSUE TITLE
Director section sometimes list more people than just the director

### ISSUE THREAD

**Issue by ChristopherAlan** [malinkas86/DoIOwnIt#10]
Take a look at Logan for example. Both IMDb and tmdb list James Magnigold as the director but the app list 11 separate people? ...

**Comment by malinkas86** [comment]
There could be multiple directors. It's outta my control

**Comment by ChristopherAlan** [comment]
Yes but I know this particular movie doesn't have 11 directors. There must be something in the tmdb API call.
